lib/messenger.rb

```diff
@@ -0,0 +1,18 @@
+require 'dotenv'
+require 'twilio-ruby'
+
+class Messenger
```

Comment:
What are you making instances of? Messenger? Or messages? 

Beware class names ending in 'er' or 'or', they are quite often a sign that the abstraction might be a bit off. This is a class that builds messages - it should be called Message, TextMessage or SMS.